### DISCUSSION TITLE
[QA] Why is InternLM2-Chat-SFT based on InternLM2-Base instead of InternLM2?

### DISCUSSION THREAD

**Discussion by underspirit** [InternLM/InternLM#606] [Q&A] [answered]
### Describe the question.

InternLM2 is an enhanced model based on InternLM2-Base, and its capabilities should be better in many domain. Why isn't the subsequent SFT model based on it?

**Comment by ZwwWayne** [comment, marked as answer]
This is simply due to the time issue. InternLM2 and InternLM2-Chat are trained parallelly due to limited time for release. Furthermore, InternLM2 and InternLM2-Chat are optimized for different capability dimensions as you can see from the evaluation results.
